whois-query/src/main/java/net/ripe/db/whois/query/acl/AccountingIdentifier.java

```diff
@@ -6,10 +6,12 @@ public class AccountingIdentifier {
 
     final private InetAddress remoteAddress;
     final private String userName;
+    private final String ssoToken;
```

Comment:
I decided to add the ssoToken coming from the request header in here, as an extra user identification step

There is another option to just keep the userName (as it is) and call the auth service to get the related ssoToken (it is 1 to 1 relation). However, this would allow the user to bypass the counting, putting a wrong email-uuid relation...(we would be ignoring the UUID from the request).

This is needed because in the Auth Index table (which I need to use because of task requirement) we are storing "SSO " as value for Auth: attribute. So for fetching the maintainters belonging to that user we need the UUID and not the email.

Reply:
makes sense. Here we only put the credentials that we are accounting for.
In this way looks like that we are accounting for an extra auth method called ssoToken...

In my last change I merge UserName with the uuid...so there are still just 2 accounting methods but the userName is now ssoUser which contains both the uuid and the userName. Hope it is clear now